src/modules/trading/components/trading--form/trading--form.styles.less

```diff
@@ -63,31 +77,47 @@
 }
 
 .TradingForm__error_message_container {
-  background: rgba(255, 39, 39, 0.2);
-  border: 1px solid rgba(255, 39, 39, 0.2);
+  background: @color-error-message-bright-fade;
+  border: 1px solid @color-error-text;
```

Comment:
rem?

Reply:
I don't see an alternative for 1px in the code. Not sure what 1px is in rem.